packages/instui-cli/README.md

```diff
@@ -38,10 +32,10 @@ instui create component
 
 - **create component** | Create an Instructure UI component. DEPRECATED, will be removed in InstUI v9.
 - **create package** | Create an Instructure UI package. DEPRECATED, will be removed in InstUI v9.
+- **create-from-template** | Copy template source file or directory to a specified destination and replace variables with designated values.
```

Comment:
We were having a discussion with Balazs about the `create-from-template` command here. Should this be deprecated as well?
@matyasf